### PR TITLE
Week2-homework

### PR DESCRIPTION
**Pull request by Nihan2019**
I don`t know will it work?
[week2homework1](https://github.com/Nihan2019/HackYourFuture-Javascript2/tree/master/Week2/homework)

### CODE REVIEW

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,135 @@
...
+    const newList = document.createElement("ul");
+    for (var key in bookInformations) {
+      const titleText = bookInformations[key].title;
+      const authorText = bookInformations[key].author;
+      const languageText = bookInformations[key].language;
+      const titleElement = document.createElement("h3");
+      const image = document.createElement('img');
+      image.setAttribute('src', bookCovers[key]);
+      image.setAttribute('alt', key);
+      image.setAttribute('width', '70%');
```
Comment (bitsagarob) [inline Week1/homework/app.js]: don't set style from JS

File: Week1/homework/app.js [hunk 2]
```diff
@@ -1,11 +1,135 @@
...
+  }
+  document.body.append(generateList());
+
+ 
+  
+  const imageOver = document.querySelector('header');
+  imageOver.addEventListener("mouseover", mouseOver);
+  imageOver.addEventListener("mouseout", mouseOut);
+  function mouseOver() {
+    imageOver.style.color = 'blue';
```
Comment (bitsagarob) [inline Week1/homework/app.js]: nice!

**Comment by bitsagarob** [comment]
this is week 1 homework?
You have linked another branch of the week 2 homework.
you made it good, nice work!
try to create a new pull request from you week 2 homework branch, that only contains the week 2 homework files and nothing of week 1 homework.

**Comment by Nihan2019** [comment]
Couldn`t you get my week2 homework?It`s in week2 branch.

**Comment by Nihan2019** [comment]
I think my pull request has some problems.I don`t know that you can  get it.

File: Week2/homework/maartjes-work.js [hunk 3]
```diff
@@ -55,7 +55,6 @@ function computeEarnings(tasks, hourlyRate) {
   return totalSum.toFixed(2);
   console.log(tasks, hourlyRate);
```
Comment (ldealmei) [inline Week2/homework/maartjes-work.js:56]: Will this statement ever run?

File: Week2/homework/array-exercises.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+console.log(firstElement(myArray));
+```
+---
+
+//Passing a parameter 'n' will return the first 'n' elements of the array.
+```js
+const myArray = [0,1,2,3,4,5,6];
+
+const firstnElements = (arr,n) => arr.slice(0,n);
+console.log(firstnElements(myArray,4));
```
Comment (ldealmei) [inline Week2/homework/array-exercises.js]: Try combining the `firstElement` and the `firstnElements` into a single function by using a defualt parameter.
Why is your code in Markdown?

File: Week2/homework/maartjes-work.js (hunk 3 above)
Comment (Nihan2019) [reply Week2/homework/maartjes-work.js:56]: Yeah,it works.

File: Week2/homework/array-exercises.js (hunk 4 above)
Comment (Nihan2019) [reply Week2/homework/array-exercises.js]: I don`t know why I did it.I think i did it  by using a default parameter.Thank you for review.

**Pull request closed without merging**